paddle/fluid/framework/section_worker.cc

```diff
@@ -30,6 +30,45 @@ void SectionWorker::Initialize(const TrainerDesc &desc) {
   for (auto &op_desc : program_->Block(0).AllOps()) {
     ops_.push_back(OpRegistry::CreateOp(*op_desc));
   }
+
+  // if not 1F1B scheduler
+  if (schedule_mode_ != 1) return;
+
+  for (auto &op : ops_) {
+    if (!op->HasAttr("pipeline_send_var")) continue;
```

Comment:
Do we have another way to distinguish whether the input of ops (send_v2 or partial_send op) is the variable to send? I mean it is discouraged to set variable name as attribute. I prefer to add a `bool` attribute.

Reply:
Done. Remove `pipeline_send_var` attr. Find the backward send var directly.